package.json

```diff
@@ -20,8 +20,13 @@
     "validate:templates": "bash ./scripts/validate-templates.sh"
   },
   "dependencies": {
+    "@aws-sdk/client-sso-oidc": "3.629.0",
```

Comment:
can these be added to .yarnrc.yml as in here: https://github.com/govuk-one-login/txma-event-self-serve/blob/4fcc551e158cd2d4a9e74c04181ebe63338b731f/.yarnrc.yml#L7 as thats better way than adding to dependencies

Reply:
these changes were moved to another branch but because we use yarn workspaces, it doesnt seem to like the changes in `.yarnrc.yml` so im keeping it in the root package.json